earth_enterprise/src/third_party/apache2/apache-ge-configure.patch

```diff
@@ -1,11 +0,0 @@
---- httpd-2.4.39-orig/configure	2019-03-27 15:05:58.000000000 +0000
```

Comment:
The end result of the configure command is slightly different without this patch, even with `--enable-socache-shmcb` set. With the patch applied, `socache_shmcb` is added to `ENABLED_DSO_MODULES`. Without it, it is only added to `DSO_MODULES`. In the SConscript, notice that `--enable-socache-shmcb` was already set on line 100, so removing this patch did change something.